hackathon/urls.py

```diff
@@ -1,13 +1,14 @@
 from django.urls import path
-from .views import (
-    HackathonListView, create_hackathon, update_hackathon, delete_hackathon, judging
-)
+from .views import *
```

Comment:
I would prefer to be more explicit in importing the views.

Reply:
Apologies, I thought we were using _all_ the views from that file and in an order to keep code condensed I just used the all selector. I do see now that we are not using all and have reverted.